.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * This is a workaround to allow ESLint to resolve plugins that were installed
+ * by an external config: https://github.com/eslint/eslint/issues/3458
+ */
+require('@rushstack/eslint-patch/modern-module-resolution')
```

Comment:
Why did you put this here instead of in @uniswap/eslint-config?
[The docs say either will work.](https://www.npmjs.com/package/@rushstack/eslint-patch#:~:text=For%20an%20even%20leaner%20setup%2C%20%40your%2Dcompany/eslint%2Dconfig%20can%20provide%20the%20patch%20as%20its%20own%20dependency.%20See%20%40rushstack/eslint%2Dconfig%20for%20a%20real%20world%20example%20and%20recommended%20approach.)

Reply:
> For an even leaner setup, @your-company/eslint-config can provide the patch as its own dependency. 

Good call! I guess I didn't see it initially. There might be some additional changes anyway, so I will release a new version and update this PR accordingly.